server/gfmd/user.c

```diff
@@ -84,6 +91,34 @@ grpassign_add_group(struct group_assignment *ga)
 	u->groups.group_prev = ga;
 }
 
+gfarm_error_t
```

Comment:
the limit_check and the uap parameter are unnecessary,
please change this definition to:
```
gfarm_error_t
user_auth_id_moidfy(struct user *,u const char *auth_method, const char *auth_user_id)
```